weave-js/src/common/components/elements/ModifiedDropdown.tsx

```diff
@@ -425,7 +422,17 @@ const ModifiedDropdown: FC<ModifiedDropdownProps> = React.memo(
       ) : (
         <>{children}</>
       );
-
+    const selectedOption = propsOptions.find(opt => opt.value === value);
+    const renderTrigger = () => {
+      if (searchQuery) {
+        return '';
+      }
```

Comment:
**question:** what does this do?

Reply:
This makes the option hidden when the user is typing a search query to avoid both text showing up at once:
<img width="536" alt="Screenshot 2024-05-21 at 12 12 49 PM" src="https://github.com/wandb/weave/assets/126013019/d4437ebf-4ea5-4e59-8ed8-b18ee7edd079">